lib/worker.js

```diff
@@ -274,16 +271,16 @@ class AssetComputeWorker {
             // call client-provided callback to transform source into 1 rendition
             await renditionsCallback(this.source, this.renditions, this.directories.out, this.params);
 
-            timer.end();
-            console.log(`processing finished successfully after ${timer.durationSec()} seconds`);
+            this.timers.callbackProcessing = callbackTimer.end();
```

Comment:
btw, `timer.end()`, `.durationSec()`, and `.duration()` all end the timer so I could remove a lot of lines